usrp.c

```diff
@@ -101,7 +101,7 @@ void usrp_list(void) {
         if (type == NULL || serial == NULL)
             continue;
 
-        printf("interface {value=usrp-%s-%s}{display=ICE9 Bluetooth%s}\n", product, serial,
```

Comment:
you might want to update `usrp_find_serial()` - this fails for me since it passes `serial=B200mini (xxxxxx),num_recv_frames=1024` as the opening arguments to libuhd (the original code also fails because it looks for the fist `-` instead of the last `-`).